src/pages/addPosts.vue

```diff
@@ -47,13 +52,30 @@
 <script setup>
 import { createPostApi } from '../composables/createPost.js'
 import { requiredField } from "../composables/validationRules"
+import { ref } from 'vue';
 import sucessfullModal from '@/components/sucessfullModal.vue';
+import { onMounted, computed } from 'vue';
 import { useRouter } from 'vue-router';
+const userListLoaded = ref(false);
 const router = useRouter();
-const { postDetails, createPost, isLoading, sucessModal , hasErrors } = createPostApi()
+const { postDetails, createPost, isLoading, sucessModal, hasErrors } = createPostApi()
+import { postStore } from '../store/post-store.js'
+const { getUsersList } = postStore();
```

Comment:
create a user store, getUsersList should not be in postStore @ShivanshSinghnnovative